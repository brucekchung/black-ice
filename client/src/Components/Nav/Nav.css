
.Nav-bar {
  background-color: #222;
  display: flex;
  justify-content: space-around;
  margin: auto;
  padding: 20px 100px 0 100px;
}

.Nav-bar a {
  border-bottom: 3px solid #eef0f2;
  color: #eef0f2;
  font-size: 1.5rem;
  font-weight: 400;
  opacity: 0.8;
  padding: 10px 0;
  text-decoration: none;
}

.Nav-bar a:hover,
.Nav-bar .selected {
  border-bottom: 3px solid #0892a5;
  color: #0892a5;
  opacity: 1.0;
}
