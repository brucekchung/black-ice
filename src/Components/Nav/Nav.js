import React from 'react'
import { NavLink } from 'react-router-dom'

const Nav = () => (
  <div className='Nav-bar'>
    <NavLink className="Nav" exact to='/'>Login</NavLink>
    <NavLink className="Nav" to='/Landing'>Landing</NavLink>
    <NavLink className="Nav" to='/DataEntry'>DataEntry</NavLink>
    <NavLink className="Nav" to='/Reports'>Reports</NavLink>
    <NavLink className="Nav" to='/ViewAll'>EditData</NavLink>
  </div>
)

export default Nav
