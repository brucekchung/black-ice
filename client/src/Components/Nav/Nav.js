import React from 'react'
import { NavLink } from 'react-router-dom'
import './Nav.css'

const Nav = () => (
  <div className='Nav-bar'>
    <NavLink className="Nav" activeClassName='selected' exact to='/'>Home</NavLink>
    <NavLink className="Nav" activeClassName='selected' to='/DataEntry'>DataEntry</NavLink>
    <NavLink className="Nav" activeClassName='selected' to='/Reports'>Reports</NavLink>
    <NavLink className="Nav" activeClassName='selected' to='/ViewAll'>EditData</NavLink>
  </div>
)

export default Nav
