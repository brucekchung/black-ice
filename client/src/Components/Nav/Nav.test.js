import React from 'react'
import Nav from './Nav'
import { shallow } from 'enzyme'

describe('Nav', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Nav />)
  })

  it('should match snapshot upon load', () => {
    expect(wrapper).toMatchSnapshot()
  })

})