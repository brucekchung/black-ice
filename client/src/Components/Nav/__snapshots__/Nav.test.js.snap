// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match snapshot upon load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="Nav"
          exact={true}
          to="/"
>
          Login
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="Nav"
          to="/Landing"
>
          Landing
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="Nav"
          to="/DataEntry"
>
          DataEntry
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="Nav"
          to="/Reports"
>
          Reports
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="Nav"
          to="/ViewAll"
>
          EditData
</NavLink>,
      ],
      "className": "Nav-bar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Login",
          "className": "Nav",
          "exact": true,
          "to": "/",
        },
        "ref": null,
        "rendered": "Login",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Landing",
          "className": "Nav",
          "to": "/Landing",
        },
        "ref": null,
        "rendered": "Landing",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "DataEntry",
          "className": "Nav",
          "to": "/DataEntry",
        },
        "ref": null,
        "rendered": "DataEntry",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "Reports",
          "className": "Nav",
          "to": "/Reports",
        },
        "ref": null,
        "rendered": "Reports",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": "EditData",
          "className": "Nav",
          "to": "/ViewAll",
        },
        "ref": null,
        "rendered": "EditData",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="Nav"
            exact={true}
            to="/"
>
            Login
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="Nav"
            to="/Landing"
>
            Landing
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="Nav"
            to="/DataEntry"
>
            DataEntry
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="Nav"
            to="/Reports"
>
            Reports
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="Nav"
            to="/ViewAll"
>
            EditData
</NavLink>,
        ],
        "className": "Nav-bar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Login",
            "className": "Nav",
            "exact": true,
            "to": "/",
          },
          "ref": null,
          "rendered": "Login",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Landing",
            "className": "Nav",
            "to": "/Landing",
          },
          "ref": null,
          "rendered": "Landing",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "DataEntry",
            "className": "Nav",
            "to": "/DataEntry",
          },
          "ref": null,
          "rendered": "DataEntry",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "Reports",
            "className": "Nav",
            "to": "/Reports",
          },
          "ref": null,
          "rendered": "Reports",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": "EditData",
            "className": "Nav",
            "to": "/ViewAll",
          },
          "ref": null,
          "rendered": "EditData",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
